src/components/Navbar/styles.module.scss

```diff
@@ -59,3 +60,16 @@ $bg-color: $dark-gray;
 .username {
   margin-right: $default-padding;
 }
+
+.reviewerToolsLink {
+  &,
+  &:active,
+  &:hover {
+    color: $white;
+    text-decoration: underline;
+  }
+
+  svg {
```

Comment:
We had a rule on `addons-frontend` to always use a css class when selecting elements. I think it's a good rule to follow because subtle performance problems can creep in without it.